vite.config.mjs

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'vite';
+import { extensions, classicEmberSupport, ember } from '@embroider/vite';
+import { babel } from '@rollup/plugin-babel';
+import { scopedCSS } from 'ember-scoped-css/vite';
+
+export default defineConfig({
+  plugins: [
+    classicEmberSupport(),
+    ember(),
+    // extra plugins here
+    babel({
+      babelHelpers: 'runtime',
+      extensions,
+    }),
+    scopedCSS({
+      layerName: 'scoped-components',
+    }),
```

Comment:
As mentioned in [ember-scoped-css docs](https://github.com/soxhub/ember-scoped-css?tab=readme-ov-file#configuration), `scopedCSS` plugin is required. The `layerName` configuration options now goes there.